src/components/main_compo_4.html

```diff
@@ -1 +1,96 @@
+            document.getElementById(ID_BTN).style.backgroundColor="#EBCB80";
+            document.getElementById(ID_BTN).style.color="#4B4B4B";
+            document.getElementById("btn_pl2").style.backgroundColor ="transparent";
+            document.getElementById("btn_pl2").style.color ="black";
+            document.getElementById("btn_pl3").style.backgroundColor ="transparent";
+            document.getElementById("btn_pl3").style.color ="black";
+            document.getElementById("plan1").style.padding = "10% 22%"
+            document.getElementById("plan3").style.padding = "5% 2% "
+            document.getElementById("plan2").style.padding = "5% 2% "
+        }
+        function gold(ID_BTN){
+            document.getElementById("plan3").style.zIndex =0;
+            document.getElementById("plan3").style.transform= "translateY(-10px)";
+            document.getElementById("plan1").style.zIndex =1;
+            document.getElementById("plan1").style.transform= "translateY(5px)";
+            document.getElementById("plan2").style.zIndex =2;
+            document.getElementById("plan2").style.transform= "translateY(27px)";
+            document.getElementById(ID_BTN).style.backgroundColor="#EBCB80";
+            document.getElementById(ID_BTN).style.color="#4B4B4B";
+            document.getElementById("btn_pl1").style.backgroundColor ="transparent";
+            document.getElementById("btn_pl1").style.color ="black";
+            document.getElementById("plan1").style.padding = "10% 17%"
+            document.getElementById("plan3").style.padding = "3% 1% "
+            document.getElementById("plan2").style.padding = "5% 8% "
+            document.getElementById("btn_pl3").style.backgroundColor ="transparent";
+            document.getElementById("btn_pl3").style.color ="black";
+        }
+        function plantinum(ID_BTN){
+            document.getElementById("plan1").style.zIndex =0;
+            document.getElementById("plan1").style.transform= "translateY(-10px)";
+            document.getElementById("plan2").style.zIndex =1;
+            document.getElementById("plan2").style.transform= "translateY(15px)";
+            document.getElementById("plan3").style.zIndex =2;
+            document.getElementById("plan3").style.transform= "translateY(30px)";
+            document.getElementById(ID_BTN).style.backgroundColor="#EBCB80";
+            document.getElementById(ID_BTN).style.color="#4B4B4B";
+            document.getElementById("btn_pl1").style.backgroundColor ="transparent";
+            document.getElementById("btn_pl1").style.color ="black";
+            document.getElementById("plan1").style.padding = "5% 8%"
+            document.getElementById("plan3").style.padding = "5% 5% "
+            document.getElementById("plan2").style.padding = "5% 6% "
+            document.getElementById("btn_pl2").style.backgroundColor ="transparent";
+            document.getElementById("btn_pl2").style.color ="black";

```

Comment:
why u use JS here? We are practicing html css, and it is not necessary to use js.

Reply:
I use js for onclick animation.

Reply:
no, it's not necessary. you should do everything in html css.